.github/workflows/signsbom.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: SignSBOM
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - ".github/workflows/signsbom.yml"
+      - "cyclonedx-lib/**"
+
+# Cancel existing runs if user makes another push.
+concurrency:
+  group: "${{ github.ref }}"
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  test_sbom_sign:
+    name: sign_sbom
+    runs-on: ubuntu-latest
+    container:
+      image: adoptopenjdk/centos7_build_image
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      # Build with jdk17 to ensure TemurinSignSBOM meets min compatibility
+      - uses: actions/setup-java@1df8dbefe2a8cbc99770194893dd902763bee34b
```

Comment:
What version is that for set-up java?  Can we add the comment on the end?

Reply:
@julian55455 v3.10.0
https://github.com/actions/setup-java/commit/1df8dbefe2a8cbc99770194893dd902763bee34b